docs/APIs/storage/block_device.md

```diff
@@ -28,6 +28,14 @@ Block devices indicate their block sizes through the `get_read_size`, `get_progr
 
 As a rule of thumb, you can use the erase size for applications that use a single block size (for example, the FAT file system).
 
+## Utility block devices
+
+mbed OS contains several utility block devices to give you better control over how storage is allocated.
```

Comment:
I suggest changing "how storage is allocated" to "the allocation of storage" to eliminate the passive voice.